.github/scripts/update-release-notes.js

```diff
@@ -0,0 +1,165 @@
+    method : 'GET',
+    headers: {
+        'Content-Type': 'application/json',
+        'User-Agent': 'server-side',
+      }
+};
+
+var reqGet = https.request(optionsget, function(res) {
+    // console.log("statusCode: ", res.statusCode);
+ 
+    res.on('data', function(d) {
+        str = new TextDecoder("utf-8").decode(d)
+        responseJson = JSON.parse(str)
+
+        responseJson.response.docs.forEach(element => {
+            if (isEarlierThan24Hours(element.timestamp) == false) {
+                if (element.a == 'sdk-bom') {
+                    const version = element.v
+
+                    fetchReleaseNotes('adobe','aepsdk-commons',`bom-${version}`, (ios1086DateStr, releaseNotes) => {
+                        const date = new Date(ios1086DateStr)
+                        const month = date.getMonth()
+                        const day = date.getDate()
+                        const year = date.getFullYear()
+                        const monthNames = ["Jan", "Feb", "Mar", "Apr", "May", "June", "July", "Aug", "Sept", "Oct", "Nov", "Dec"];
+                        const shortMonth = monthNames[month]
+                        //## Aug 8, 2023
+                        console.error(`## ${shortMonth} ${day}, ${year}`);
+                        //### Android BOM 2.3.1
+                        console.error(`### Android BOM  ${version}`);
+                        console.error(releaseNotes)
+
+                        const releaseMdPath = "./src/pages/documentation/release-notes/index.md";
+
+                        const releaseNotesHeader = "# Release notes";
+                        // Read the contents of the markdown file.
+                        const contentArray = fs.readFileSync(releaseMdPath, "utf8").toString().split("\n");
+                         // Find the index of the release notes header.
+                        const releaseNotesHeaderIndex = contentArray.indexOf(releaseNotesHeader);
+                        console.error(releaseNotesHeaderIndex)
+
+                        if (releaseFileContainsLineStartWith(`### Android BOM  ${version}`) == true) {
+                            console.error("already updated")
+                            return
+                        }
+                        contentArray.splice(releaseNotesHeaderIndex + 1,0,"",`## ${shortMonth} ${day}, ${year}`,"",`### Android BOM  ${version}`, "",updateBOMReleaseNotesForAdobeIO(releaseNotes));
+                        fs.writeFile(releaseMdPath, contentArray.join("\n"), function (err) {
+                            if (err) return console.log(err);
+                          });
+                    })
+                }
+                
+            }
+        });
+    });
+ 
+});
+ 
+reqGet.end();
+reqGet.on('error', function(e) {
+    console.error(e);
+});
+
+}
+
+function updateBOMReleaseNotesForAdobeIO(releaseNotesString) {
+    releaseNotesAsArray = releaseNotesString.split("\n")
```

Comment:
Why don't you just return the text from https://github.com/adobe/aepsdk-commons/releases/tag/bom-2.2.0 so that we don't have to do add additional logic here.

Reply:
I hope we can do that. The Adobe IO component `<Accordion>` required here is not supported by the GitHub markdown syntax.